Makefile

```diff
@@ -117,6 +118,10 @@ else ifeq ($(GRUCODE),f3dzex) # Fast3DZEX (2.08J / Animal Forest - Dōbutsu no M
 else ifeq ($(GRUCODE),super3d) # Super3D
   $(warning Super3D is experimental. Try at your own risk.)
   DEFINES += SUPER3D_GBI=1 F3D_NEW=1
+else ifeq ($(GRUCODE),f3dex3) # Fast3DEX3
+  DEFINES += F3DEX_GBI_3=1 F3DEX_GBI_SHARED=1
+  $(warning F3DEX3 is experimental, and as of February 4th, 2024, only fully works on a real Nintendo 64 or with low level emulation (LLE) (i.e. NOT PROJECT64 1.6). Try at your own risk and please inform your audience to avoid compatibility problems.)
+  $(warning Failure to change emulator settings CAN CAUSE YOUR GAME TO CRASH!)
```

Comment:
Nit: These conditions should appear in the same order as the listed options above

Reply:
Was also referring to the comments